integration-tests/docker/docker-compose.query-error-test.yml

```diff
@@ -78,7 +78,7 @@ services:
       druid-it-net:
         ipv4_address: 172.172.172.14
     ports:
-      - 8084:8083
+      - 8086:8083
```

Comment:
Is this an actual change in port assignments? If so, then we're changing the host port from 8084 to 8086. (The port in the container is 8083). The tests have to know to use this new port. Did we make the matching changes elsewhere?

Reply:
The host port mapping from the historical container is not used by the `query-error` or `query-retry` group of tests. If they do, they'll result in an error whether the mapped port is 8084 or 8086 due to this hardcoding. https://github.com/apache/druid/blob/698670c88e3993be971d5c7dda5fed281e32f6de/integration-tests/src/main/java/org/apache/druid/testing/DockerConfigProvider.java#L196-L199 Maybe we can update this in future to take the host port of each service as input from a config file. For now 8084 can't be used as it's being used by mono service in Github hosted runners.

Reply:
Please add a comment to the file to explain this fact so we don't get confused in the future. To avoid triggering an entire new build, just make a note to add the comment in your next PR.